frontend/src/js/controllers/challengeListCtrl.js

```diff
@@ -52,6 +52,11 @@
                     } else {
                         vm.currentList[i].isLarge = "";
                     }
+                    var timezone=moment.tz.guess();
```

Comment:
```suggestion

```
the timezone of the client can be calculated only once, not needed to be done repeatedly. Moving out of loop.